src/init.cpp

```diff
@@ -574,6 +575,10 @@ void SetupServerArgs()
     gArgs.AddArg("-zmqpubrawtx=<address>", "Enable publish raw transaction in <address>", false, OptionsCategory::ZMQ);
     gArgs.AddArg("-zmqpubrawtxlock=<address>", "Enable publish raw transaction (locked via InstantSend) in <address>", false, OptionsCategory::ZMQ);
     gArgs.AddArg("-zmqpubrawtxlocksig=<address>", "Enable publish raw transaction (locked via InstantSend) and ISLOCK in <address>", false, OptionsCategory::ZMQ);
+    gArgs.AddArg("-zmqpubhashblockhwm=<n>", strprintf("Set publish hash block outbound message high water mark (default: %d)", CZMQAbstractNotifier::DEFAULT_ZMQ_SNDHWM), false, OptionsCategory::ZMQ);
+    gArgs.AddArg("-zmqpubhashtxhwm=<n>", strprintf("Set publish hash transaction outbound message high water mark (default: %d)", CZMQAbstractNotifier::DEFAULT_ZMQ_SNDHWM), false, OptionsCategory::ZMQ);
+    gArgs.AddArg("-zmqpubrawblockhwm=<n>", strprintf("Set publish raw block outbound message high water mark (default: %d)", CZMQAbstractNotifier::DEFAULT_ZMQ_SNDHWM), false, OptionsCategory::ZMQ);
+    gArgs.AddArg("-zmqpubrawtxhwm=<n>", strprintf("Set publish raw transaction outbound message high water mark (default: %d)", CZMQAbstractNotifier::DEFAULT_ZMQ_SNDHWM), false, OptionsCategory::ZMQ);
```

Comment:
The idea in the original PR was to fix the issue with messages being dropped when zmq hits the limit. In bitcoin the issue was related to high tx rates. In our case it can also be related to at least islocks and governance votes. I think we should apply similar changes and add `*hwm` args for Dash specific messages. Should probably do this for every message not only islock/govvote related ones since bitcoin went with blocks too (simply for consistency sake I guess).

Reply:
@dzutte-cpp do you have any idea when you will get around to this?

Reply:
@PastaPastaPasta I may need some time to figure this out, let me convert this PR into a draft for the time being.
Alternatively, if the feature is needed sooner, I can hand it over to someone - don't want to keep folks blocked.
Let me know which way works better.

Reply:
No worries :) I just wanted to make sure this PR wasn't being forgotten.